Library/Formula/libquantum.rb

```diff
@@ -0,0 +1,35 @@
+class Libquantum < Formula
+  desc "C library for the simulation of quantum mechanics"
+  homepage "http://www.libquantum.de/"
+  url "http://www.libquantum.de/files/libquantum-1.0.0.tar.gz"
```

Comment:
Any reason to use the `1.0.0` release instead of the `1.1.1`; which has been released in 2013 according to the homepage?